psalm.xml

```diff
@@ -50,6 +50,15 @@
                 <directory name="*"/>
             </errorLevel>
         </RedundantCastGivenDocblockType>
+        <!--
```

Comment:
I'd put this into the code via `@psalm-suppress` as that would be reported once its fixed in upstream and thus we are able to remove that suppression.
Suppressions in the psalm config are not reported and thus would suppress this issue for like forever which might introduce problems somewhen in the future.

Reply:
Makes sense, I changed this. Also made a PR for changing this in psalm and referenced it in the docstring: https://github.com/vimeo/psalm/pull/10862